CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionExtensions.cs

```diff
@@ -93,8 +93,11 @@ public static uint CalculateFee(this Transaction transaction, uint? a = null, ui
             return ((uint)transaction.Serialize().Length * a.Value) + b.Value;
         }
 
-        public static uint CalculateAndSetFee(this Transaction transaction, uint? a = null, uint? b = null)
+        public static uint CalculateAndSetFee(this Transaction transaction, uint? a = null, uint? b = null, int mockedVKeyWitnesses = 0)
         {
+            if(mockedVKeyWitnesses > 0)
+                transaction.TransactionWitnessSet.VKeyWitnesses.CreateMocks(mockedVKeyWitnesses);
+            
```

Comment:
maybe we should add some code after the fee calculation to remove the mocked witnesses

Reply:
not remove all the mocks, i mean just if they were put as a parameter for the calculateFee method

Reply:
oh, i see... we are removing all the mocks anyway at the moment... nevermind